### PR TITLE
Added documentation for ACE support for imported RKE2/K3S clusters

### LINKED ISSUE #3612: [Rancher2] Add documentation for ACE support for imported RKE2/K3S clusters
**Request Summary:**

The plan is to add support for ACE for imported RKE2/K3S clusters in Rancher v2.6.3. This support includes manual steps that the user would have to take on the downstream cluster. The documentation should describe these steps in detail, emphasizing the order in which the steps should be taken.

1. The manual steps should be taken fist:
	- Create a file at /var/lib/rancher/{rke2,k3s}/kube-api-authn-webhook.yaml with some very simple contents
	- Add to the config file (or create one if it doesn’t exist):
	  ```
	  kube-apiserver-arg:
	  	authentication-token-webhook-config-file=/var/lib/rancher/{rke2,k3s}/kube-api-authn-webhook.yaml
	  ```
	- Restart with `sudo systemctl`
3. Turn on ACE from the Rancher UI/API.

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3612.

References:
https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/registered-clusters/
https://rancher.com/docs/rancher/v2.6/en/overview/architecture-recommendations/#architecture-for-an-authorized-cluster-endpoint
https://rancher.com/docs/rancher/v2.6/en/overview/architecture/#4-authorized-cluster-endpoint

### CODE REVIEW

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md [hunk 1]
```diff
@@ -147,8 +148,49 @@ If the cluster becomes stuck in upgrading, restart the `system-upgrade-controlle
 
 To prevent issues when upgrading, the [Kubernetes upgrade best practices](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/) should be followed.
 
-
-
+# ACE Support for RKE2 and K3s Clusters
```
Comment (btat) [inline content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md:151]: ACE isn't a common acronym so we should use the full form.

```suggestion
# Authorized Cluster Endpoint Support for RKE2 and K3s Clusters
```

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ The control that Rancher has to manage a registered cluster depends on the type
 - [Management Capabilities for Registered Clusters](#management-capabilities-for-registered-clusters)
 - [Configuring K3s Cluster Upgrades](#configuring-k3s-cluster-upgrades)
 - [Debug Logging and Troubleshooting for Registered K3s Clusters](#debug-logging-and-troubleshooting-for-registered-k3s-clusters)
+- [ACE Support for RKE2 and K3s Clusters](#ace-support-for-rke2-and-k3s-clusters)
```
Comment (btat) [inline content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md:15]: ACE isn't a common acronym so we should use the full form.

```suggestion
- [Authorized Cluster Endpoint Support for RKE2 and K3s Clusters](#ace-support-for-rke2-and-k3s-clusters)
```

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md [hunk 3]
```diff
@@ -147,8 +148,49 @@ If the cluster becomes stuck in upgrading, restart the `system-upgrade-controlle
...
 
-
-
+# ACE Support for RKE2 and K3s Clusters
+
+_Available as of v2.6.3_
+
+**Authorized Cluster Endpoint** (ACE) support has been added for registered RKE2 and K3s clusters. This support includes manual steps you will perform on the downstream cluster to enable the ACE. For additional information on the ACE, click [here]({{<baseurl>}}/rancher/v2.6/en/cluster-admin/cluster-access/ace/).
+
+> **Notice:**
```
Comment (btat) [inline content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md:157]: Maybe stick with `Note` for consistency with the rest of the docs.

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md [hunk 4]
```diff
@@ -147,8 +148,49 @@ If the cluster becomes stuck in upgrading, restart the `system-upgrade-controlle
...
+        
+1. Add the following to the config file (or create one if it doesn’t exist); note that the default location is `/etc/rancher/{rke2,k3s}/config.yaml`:
+
+        kube-apiserver-arg:
+            - authentication-token-webhook-config-file=/var/lib/rancher/{rke2,k3s}/kube-api-authn-webhook.yaml
+        
+1. Finally, run the following commands:
+
+  - `sudo systemctl stop {rke2,k3s}-server`
+  - `sudo systemctl start {rke2,k3s}-server`
```
Comment (btat) [inline content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md:193]: These should be put into one code block

File: content/rancher/v2.6/en/overview/architecture-recommendations/_index.md [hunk 5]
```diff
@@ -109,3 +109,5 @@ For more best practices for downstream clusters, refer to the [production checkl
 If you are using an [authorized cluster endpoint,]({{<baseurl>}}/rancher/v2.6/en/overview/architecture/#4-authorized-cluster-endpoint) we recommend creating an FQDN pointing to a load balancer which balances traffic across your nodes with the `controlplane` role.
 
 If you are using private CA signed certificates on the load balancer, you have to supply the CA certificate, which will be included in the generated kubeconfig file to validate the certificate chain. See the documentation on [kubeconfig files]({{<baseurl>}}/rancher/v2.6/en/cluster-admin/cluster-access/kubectl/) and [API keys]({{<baseurl>}}/rancher/v2.6/en/user-settings/api-keys/#creating-an-api-key) for more information.
+
+As of Rancher v2.6.3, ACE support is available for registered RKE2 and K3s clusters. To view the manual steps to perform on the downstream cluster to enable the ACE, click [here]({{<baseurl>}}/v2.6/en/cluster-provisioning/registered-clusters/#ace-support-for-rke2-and-k3s-clusters).
```
Comment (btat) [inline content/rancher/v2.6/en/overview/architecture-recommendations/_index.md:113]: The ACE acronym isn't defined/established on this page.

**Review by btat — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md (hunk 3 above)
Comment (jtravee) [reply content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md]: Updated.

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md (hunk 4 above)
Comment (jtravee) [reply content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md]: Updated.

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md]: Updated.

File: content/rancher/v2.6/en/overview/architecture-recommendations/_index.md (hunk 5 above)
Comment (jtravee) [reply content/rancher/v2.6/en/overview/architecture-recommendations/_index.md]: Added clarification for acronym, ty!

**Review by btat — APPROVED** [review]

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md [hunk 6]
```diff
@@ -147,8 +148,49 @@ If the cluster becomes stuck in upgrading, restart the `system-upgrade-controlle
...
 
+Authorized Cluster Endpoint (ACE) support has been added for registered RKE2 and K3s clusters. This support includes manual steps you will perform on the downstream cluster to enable the ACE. For additional information on the ACE, click [here]({{<baseurl>}}/rancher/v2.6/en/cluster-admin/cluster-access/ace/).
+
+> **Note:**
+>
+> - The following steps will work on both RKE2 and K3s clusters registered in v2.6.x as well as those registered (or imported) from a previous version of Rancher with an upgrade to v2.6.x.
+>
+> - These steps will alter the configuration of the downstream RKE2 and K3s clusters and deploy the `kube-api-authn-webhook`. If a future implementation of ACE requires an update to the `kube-api-authn-webhook`, then this would also have to be done manually. For more information on this webhook, click [here]({{<baseurl>}}/rancher/v2.6/en/cluster-admin/cluster-access/ace/#about-the-kube-api-auth-authentication-webhook).
+
+###### **Manual steps to be taken on the downstream cluster to enable ACE:**
```
Comment (thedadams) [inline content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md:163]: These instructions should indicate that these manual steps only need to be taken on the control plane nodes of the downstream cluster. Also, the manual steps need to be taken on each control plane node.

**Review by thedadams — CHANGES_REQUESTED** [review]

**Review by thedadams — APPROVED** [review]

**Pull request merged**